uid-pages/page-admin-profile-list/src/adminProfileListBonitaV1.json

```diff
@@ -14013,6 +14018,15 @@
         "    isAddMembershipGroupSearchMore: function(item) {",
         "        return item.name === uiTranslate(\"Or type more...\");",
         "    },",
+        "    isAddMembershipRoleSearchMore: function(item) {",
+        "        return item.name === uiTranslate(\"Or type more...\");",
+        "    },",
+        "    getGroupDisplayName(group) {",
+        "      return group.displayName ? group.displayName : group.name;  ",
```

Comment:
Why do you have the function twice, also, the function is not useful

Reply:
One is for the group mapping itself, and one is for a group in membership mapping. Maybe I should rename the function for the group in the membership mapping.